src/PrestaShopBundle/Form/Admin/Sell/Product/BasicInformationType.php

```diff
@@ -54,11 +54,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     $this->trans('Standard product', 'Admin.Catalog.Feature') => ProductType::TYPE_STANDARD,
                     $this->trans('Pack of products', 'Admin.Catalog.Feature') => ProductType::TYPE_PACK,
                     $this->trans('Virtual product', 'Admin.Catalog.Feature') => ProductType::TYPE_VIRTUAL,
-                    $this->trans('Product with combinations', 'Admin.Catalog.Feature') => ProductType::TYPE_COMBINATION,
+                    $this->trans('Product with combinations', 'Admin.Catalog.Feature') => ProductType::TYPE_COMBINATIONS,
                 ],
                 'choice_translation_domain' => 'Admin.Catalog.Feature',
                 'attr' => [
                     'class' => 'custom-select',
+                    'data-modal-title' => $this->trans('Changing product type', 'Admin.Catalog.Notification'),
```

Comment:
@Julievrz I'd like your feedback regarding these new wordings Seems like the bot didn't detect them for some reason